provider/clickhouse.go

```diff
@@ -101,15 +102,15 @@ func (store *clickHouseOfflineStore) createTransformationName(id ResourceID) (st
 	case Transformation:
 		return GetPrimaryTableName(id)
 	case Label:
-		return "", TransformationTypeError{"Invalid Transformation Type: Label"}
+		return "", fferr.NewInvalidArgument(fmt.Errorf("invalid transformation type: Label"))
```

Comment:
Don't we have an InvalidResourceType error?

Reply:
Good call. I should use this instead. Will correct this in `sql.go`, too.